Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 173, in wrapped
    return loop.run_until_complete(inner)
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import osmnx as ox
from nrel.routee.compass.io import generate_compass_dataset
from nrel.routee.compass import CompassApp
from pathlib import Path
import os
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mImportError[0m                               Traceback (most recent call last)
Cell [0;32mIn[1], line 2[0m
[1;32m      1[0m [38;5;28;01mimport[39;00m [38;5;21;01mosmnx[39;00m [38;5;28;01mas[39;00m [38;5;21;01mox[39;00m
[0;32m----> 2[0m [38;5;28;01mfrom[39;00m [38;5;21;01mnrel[39;00m[38;5;21;01m.[39;00m[38;5;21;01mroutee[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcompass[39;00m[38;5;21;01m.[39;00m[38;5;21;01mio[39;00m [38;5;28;01mimport[39;00m generate_compass_dataset
[1;32m      3[0m [38;5;28;01mfrom[39;00m [38;5;21;01mnrel[39;00m[38;5;21;01m.[39;00m[38;5;21;01mroutee[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcompass[39;00m [38;5;28;01mimport[39;00m CompassApp
[1;32m      4[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpathlib[39;00m [38;5;28;01mimport[39;00m Path

File [0;32m/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/nrel/routee/compass/__init__.py:1[0m
[0;32m----> 1[0m [38;5;28;01mfrom[39;00m [38;5;21;01mnrel[39;00m[38;5;21;01m.[39;00m[38;5;21;01mroutee[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcompass[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcompass_app[39;00m [38;5;28;01mimport[39;00m CompassApp
[1;32m      2[0m [38;5;28;01mfrom[39;00m [38;5;21;01mnrel[39;00m[38;5;21;01m.[39;00m[38;5;21;01mroutee[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcompass[39;00m[38;5;21;01m.[39;00m[38;5;21;01mio[39;00m[38;5;21;01m.[39;00m[38;5;21;01mgenerate_dataset[39;00m [38;5;28;01mimport[39;00m generate_compass_dataset

File [0;32m/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/nrel/routee/compass/compass_app.py:7[0m
[1;32m      5[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpathlib[39;00m [38;5;28;01mimport[39;00m Path
[1;32m      6[0m [38;5;28;01mfrom[39;00m [38;5;21;01mtyping[39;00m [38;5;28;01mimport[39;00m Any, Dict, List, Union
[0;32m----> 7[0m [38;5;28;01mfrom[39;00m [38;5;21;01mnrel[39;00m[38;5;21;01m.[39;00m[38;5;21;01mroutee[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcompass[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcompass_app_py[39;00m [38;5;28;01mimport[39;00m (
[1;32m      8[0m     CompassAppWrapper,
[1;32m      9[0m )
[1;32m     12[0m [38;5;28;01mclass[39;00m [38;5;21;01mCompassApp[39;00m:
[1;32m     13[0m [38;5;250m    [39m[38;5;124;03m"""[39;00m
[1;32m     14[0m [38;5;124;03m    The CompassApp holds everything needed to run a route query.[39;00m
[1;32m     15[0m [38;5;124;03m    """[39;00m

[0;31mImportError[0m: dynamic module does not define module export function (PyInit_compass_app_py)

